src/stack.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+struct StackNode {
+    int value;
+    struct StackNode* next;
+};
+
+struct Stack {
```

Comment:
Может возникнуть вопрос: "а как тогда объявлять `Stack`, если `StackNode` у нас только в .c существует?". Можно сделать просто вот так:

```suggestion
typedef struct Stack Stack;
```